### ISSUE TITLE
Integrate Updated Dynamic Lighting

### ISSUE THREAD

**Issue by burchbobby** [SSStormy/roll20-enhancement-suite#166]
Roll20 recently released their Updated Dynamic Lighting, which adds a new panel of options in map settings, as well as for individual tokens.

The Enhancement Suite should include these new Dynamic Lighting options in the Default Token Editor.

**Comment by kenbolton** [comment]
I agree in principle, but UDL is definitely not ready for prime-time as of this writing. It is not feature-complete and remains buggy.

**Comment by SSStormy** [comment]
We're not touching the new updated dynamic lighting untill the legacy version is discontinued.
